disque/disque.go

```diff
@@ -111,14 +108,15 @@ func (c *RedisClient) Add(r AddRequest) (string, error) {
 	return id, nil
 }
 
-// SendAdd sends an ADDJOB command to disque without waiting for reply
-func (c *RedisClient) SendAdd(r AddRequest) error {
-	return c.conn.Send("ADDJOB", addArgs(r)...)
-}
-
-// FinishAdd receives all replies for SendAdd calls
-func (c *RedisClient) FinishAdd() ([]string, error) {
-	replies, err := redis.Values(c.conn.Do("")) //  flush the output buffer and receive pending replies
+// AddMulti sends multiple ADDJOB in pipeline
+func (c *RedisClient) AddMulti(rs []AddRequest) ([]string, error) {
+	for _, r := range rs {
+		if err := c.conn.Send("ADDJOB", addArgs(r)...); err != nil {
+			return nil, err
+		}
+	}
+	// flush the output buffer and receive pending replies
+	replies, err := redis.Values(c.conn.Do(""))
```

Comment:
Isn't this usually done with flush/receive? never seen this pattern. but if it works - fine!